src/microbe_stage/MicrobeHUD.cs

```diff
@@ -75,6 +78,19 @@ public override void _Ready()
         bindingModeHotkey = GetNode<ActionButton>(BindingModeHotkeyPath);
         unbindAllHotkey = GetNode<ActionButton>(UnbindAllHotkeyPath);
 
+        mouseHoverPanel.AddCategory("compounds", new LocalizedString("COMPOUNDS_COLON"));
+        mouseHoverPanel.AddCategory("species", new LocalizedString("SPECIES_COLON"));
+        mouseHoverPanel.AddCategory("chunk", new LocalizedString("FLOATING_CHUNK_COLON"));
```

Comment:
Shouldn't "chunks" be plural here. Also as these are used as keys here, I think either using an enum or a private const variable to hold the string would be better.

With the enum approach the enum needs values for all stages at once in it, but that shouldn't be much of a problem.

Reply:
This is now fixed with const, enum might feel too restrictive.